README.md

```diff
@@ -709,6 +710,20 @@ $ remme transaction get-list \
 }
 ```
 
+Get a list of transactions' identifiers:
```

Comment:
`Get a list of transactions' identifiers` > `Get a list of transactions' identifiers (can be combined with other parameters like `--limit`)`.

Reply:
@dmytrostriletskyi, done.